sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -4033,6 +4040,7 @@ _ZNK4sycl3_V15queue12get_propertyINS0_8property5queue16enable_profilingEEET_v
 _ZNK4sycl3_V15queue12get_propertyINS0_8property5queue8in_orderEEET_v
 _ZNK4sycl3_V15queue12has_propertyINS0_8property5queue16enable_profilingEEEbv
 _ZNK4sycl3_V15queue12has_propertyINS0_8property5queue8in_orderEEEbv
+_ZNK4sycl3_V15queue28ext_codeplay_supports_fusionEv
```

Comment:
Note that CI does not run the Windows variant of this test, but post-commit does. As such, when this is merged post-commit will likely start complaining. If you have a windows machine you can run tests with, it might be a good idea to give this patch a run-through with that. If not, please add the missing symbols ASAP after this has been merged.

Reply:
Thanks for pointing that out @steffenlarsen!

We would like to avoid breaking the post-commit pipeline, so I've updated the corresponding Windows symbol dump/test in 11349a7bba3738fbdf5640335872e5806365c227 as part of this PR.

Reply:
Awesome, thanks!